### ISSUE TITLE
Document Selection model for VFB 1.5 stack browser.

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB#619]
I find the selection model on the VFB 1.5 stack browser hard to work with.  Please could it be documented?   Some questions:

* (how) Can I turn hover-over display of region on?
* (how) Can I select a painted domain by using the cursor and the image?
* Finding everything with voxels at some point is useful, but seems quite slow.  Should this really be the default state?
    * This function appears to only work for single neurons, is the background in expression patterns and clone images too high for it to be viable for these too?  If so, could this be made clear somewhere?

* Selection on the stack used to mean selection in term info.  Now, to choose a neuropil domain I need to open out the tree browser.  Navigate manually to a term. Then click on the text in order for detail to be displayed.

**Comment by Robbie1977** [comment]
Hover would be causing multiple server request every time the mouse stops over the image - this would have to be limited to anatomy and still (in my opinion) is very wasteful when most of the time this isn't useful. 

Double click on the same point toggles selection on the first anatomy item.

This should work for expressions patterns too - do you have an example that doesn't? Background on expression should be thresholded to 0 and not be detected.

I can have selection open the full details however as overlapping domains (tracts etc) occur this will become less obvious to resolve. you shouldn't need to open the tree browser?

**Comment by dosumis** [comment]
Seems to be working better now than when I tested before - although maybe I've just started to work out how its meant to work .

I still think there is work to do re selection model and layout to make sure we support expected uses and do so in an intuitive way. 

1. User wants to know what region of the brain they are looking at is called along with some basic information about it (perhaps prompted by signal indicating GAL4 expression). Do they have a quick, intuitive option that gives them this functionality?

Right now they need to know to click on a brain region - in which case they will see the brain region term along with a list of other things selected for display on the stack (could be very many).  They then need to know which of the things in the selection list corresponds to the domain and select the Info button.

I think that something closer to the older model would be faster and more intuitive: We privilege the most specific painted brain regions so that they show up in float-over but when clicked display everything in the region currently loaded on the brain. (The current float-over message is too jargon-y - new users will not know what can be selected or what 'state' refers to).   Double clicking should toggle display of domain as now.  I think it should also  automatically select term info as the current site does. 

2. A user wants to choose a particular expression pattern from the viewer and then get associated info as well as a 2D maximum projection to use as a guide for navigating. 

Right now they first need to find the pattern in question by clicking on the image, then they need to know to choose info, and that they can find a maximum projection image at the bottom of the page.  Unfortunately this is not very useful for navigating except on monitors large enough to display both browser and 2D projection on the same screen.

I'm not sure how or whether the selection model can be improved here, although it will be interesting to observe how may users get it in UX testing. The remaining issue is one of layout.  Can we find a way to lay the page out so that the 2D maximum projection is displayed somewhere it can be used as a guide for navigation in the browser?
